open-company/open-company-web
# Mobile tweaks

Card: https://trello.com/c/V1JXbBC4

To test:
- open app on mobile
- do you see the search header completely hidden initially?
- go to Unread
- pull down the header
- do you NOT see the complete all button (make sure you have something in Unread or it will be hidden anyway)
- tap into the search field
- can you search?
- do you see search history?
- open app on desktop
- can you search?
- do you see search history?
- can you click on a result to open it?


To test trial expired banner:
- use an org with expired trial
- open on mobile as admin
- do you see the banner at the top?
- do you see all the copy (on iPhone X or wider devices you should see it all, on smaller ones you should see the short copy)?
- can you open the Billing panel from the banner?
- can you open the chat from the banner?
- can you not create a new post?
- now open as author/viewer
- do you see the other copy?
- can you open the chat from the banner?
- can you NOT create a post?
- expand a post
- do you see the banner, the post header and the post title with good spacing at the top?
- check also on desktop for regressions
